packages/swingset/__tests__/parse-component-path.test.ts

```diff
@@ -23,19 +23,19 @@ describe('parseComponentPath', () => {
 
     expect(result).toEqual(expectation)
   })
-  it('converts the front matter string to an object', ({ expect }) => {
+  it('Converts a 1 segment path to a navigationData object', ({ expect }) => {
     const expectation = {
+      category: 'default',
       page: 'Button',
     }
 
     const result = parseComponentPath('Button')
 
     expect(result).toEqual(expectation)
   })
-  it('Throws an error when invalid input is received', ({expect}) => {
+  it('Throws an error when too many segments are received', ({ expect }) => {
+    const input = 'edibles/fruits/berries/blueberries'
 
-    expect(() => parseComponentPath('edibles/fruits/berries/blueberries')).toThrowError()
-   
-    
+    expect(() => parseComponentPath(input)).toThrowError()
```

Comment:
💭 What do you think about opting for a snapshot here?

(You'll have to re-run `npm run test` and commit the updated result)

```suggestion
    expect(() => parseComponentPath(input)).toThrowErrorMatchingInlineSnapshot()
```